docs/source/examples/annotations.rst

```diff
@@ -28,6 +28,23 @@ Annotations
             tip_text = Text('(2, 2)').next_to(arrow.get_end(), RIGHT)
             self.add(numberplane, dot, arrow, origin_text, tip_text)
 
+.. manim:: UsefulAnnotations
+    :quality: medium
```

Comment:
The quality used for rendering the directive without options is optimized for the documentation already. Is there a reason for using `:quality: medium` here? If not, I'd suggest to remove it:

```suggestion
```

Reply:
nope, there is not :)
I just copy pasted it from another example, were we also still have a `    :quality: medium` flag